snippets/loader.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Loaders
+author: irustm
+level: beginner
+tags:
+  - tips
+---
+# Content
+
+Loader options
+
+```typescript
+import { Component, VERSION, Directive, Input, ViewContainerRef, ComponentFactoryResolver } from '@angular/core';
+
+@Component({
```

Comment:
Can we turn this in a simple structural directive with 1 example? too much stuff otherwise.